From 07551cbd18a59c9ce8ca2d0c95471b794c403735 Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko <skupriienko@anaconda.com>
Date: Thu, 13 Oct 2022 18:03:56 +0300
Subject: [PATCH] Fix leptonica

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bd2649de..e6fbc07c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -377,7 +377,6 @@ else()
   if(NOT Leptonica_FOUND)
     message(FATAL_ERROR "Cannot find required library Leptonica. Quitting!")
   endif(NOT Leptonica_FOUND)
-  include_directories(${Leptonica_INCLUDE_DIRS})
 
   # Check for optional libraries.
   if(WIN32)
-- 
2.34.1

