From dab50471ca105caaf60c6a51e3c19d97384eaa3f Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko <skupriienko@anaconda.com>
Date: Thu, 13 Oct 2022 15:07:15 +0300
Subject: [PATCH] Fix leptonica

---
 CMakeLists.txt | 20 +++++++-------------
 1 file changed, 7 insertions(+), 13 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bd2649de..fee7c313 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -363,21 +363,15 @@ if(SW_BUILD)
   sw_execute()
 else()
   find_package(PkgConfig)
-  if(BUILD_TRAINING_TOOLS AND NOT PKG_CONFIG_FOUND)
-    message(WARNING "Building of Training Tools requires PkgConfig")
-    set(BUILD_TRAINING_TOOLS OFF)
-  endif(BUILD_TRAINING_TOOLS AND NOT PKG_CONFIG_FOUND)
-  # Check for required library. option -DLeptonica_DIR=path => cmake hint where
-  # to find leptonica
-  find_package(Leptonica ${MINIMUM_LEPTONICA_VERSION} CONFIG)
-  if(NOT Leptonica_FOUND AND PKG_CONFIG_EXECUTABLE)
-    pkg_check_modules(Leptonica lept>=${MINIMUM_LEPTONICA_VERSION})
-    link_directories(${Leptonica_LIBRARY_DIRS})
+  if(PKG_CONFIG_EXECUTABLE AND NOT Leptonica_DIR)
+      pkg_check_modules(Leptonica REQUIRED lept>=${MINIMUM_LEPTONICA_VERSION})
+      link_directories(${Leptonica_LIBRARY_DIRS})
+  else()
+      find_package(Leptonica ${MINIMUM_LEPTONICA_VERSION} REQUIRED CONFIG)
   endif()
-  if(NOT Leptonica_FOUND)
-    message(FATAL_ERROR "Cannot find required library Leptonica. Quitting!")
+  if (NOT Leptonica_FOUND)
+      message(FATAL_ERROR "Cannot find required library Leptonica. Quitting!")
   endif(NOT Leptonica_FOUND)
-  include_directories(${Leptonica_INCLUDE_DIRS})
 
   # Check for optional libraries.
   if(WIN32)
-- 
2.34.1
